scripts/gulpfiles/build_tasks.js

```diff
@@ -66,7 +66,25 @@ const NAMESPACE_OBJECT = '$';
 /**
  * A list of chunks.  Order matters: later chunks can depend on
  * earlier ones, but not vice-versa.  All chunks are assumed to depend
- * on the first chunk.
+ * on the first chunk.  Properties are as follows:
+ *
+ * - .name: the name of the chunk.  Used to label it when describing
+ *   it to Closure Compiler and forms the prefix of filename the chunk
+ *   will be written to.
+ * - .entry: the source .js file which is the entrypoint for the
+ *   chunk.
+ * - .exports: a variable or property that will (prefixed with
+ *   NAMESPACE_OBJECT) be returned from the factory function and which
+ *   (sans prefix) will be set in the global scope to that returned
+ *   value if the module is loaded in a browser.
+ * - .importAs: the name that this chunk's exports object will be
+ *   given when passed to the factory function of other chunks that
+ *   depend on it.  (Needs to be distinct from .exports since (e.g.)
+ *   "Blockly.blocks.all" is not a valid variable name.)
+ * - .factoryPreamble: code to override the default wrapper factory
+ *   function preamble.
+ * - .factoryPostamble: code to override the default wrapper factory
+ *   function postabmle.
```

Comment:
TIL postamble is a word. Thanks!

Reply:
I actually had to check that myself.  I was very glad to discover that it was.